src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java

```diff
@@ -95,42 +91,29 @@ private void handle(
         Supplier<UUID> envelopeIdSupplier,
         Condition postLeaseCondition
     ) {
-        leaseAcquirer
-            .acquireFor(blobClient)
-            .ifPresentOrElse(
-                leaseClient -> {
-                    if (postLeaseCondition.isMet()) {
-                        UUID id = envelopeIdSupplier.get();
-                        try {
-                            byte[] rawBlob = downloadBlob(blobClient);
-
-                            var verificationResult = blobVerifier.verifyZip(blobClient.getBlobName(), rawBlob);
-
-                            if (verificationResult.isOk) {
-                                dispatch(blobClient, id, rawBlob);
-                            } else {
-                                reject(blobClient, id, verificationResult.error);
-                            }
-                        } catch (Exception exception) {
-                            handleError(id, blobClient, exception);
-                        } finally {
-                            tryToReleaseLease(leaseClient, blobClient.getBlobName(), blobClient.getContainerName());
-                        }
-                    } else {
-                        logger.info(
-                            "Skipping file: {}. File name: {}, container: {}",
-                            postLeaseCondition.getMessage(),
-                            blobClient.getBlobName(),
-                            blobClient.getContainerName()
-                        );
-                    }
-                },
-                () -> logger.info(
-                    "Cannot acquire a lease for blob - skipping. File name: {}, container: {}",
-                    blobClient.getBlobName(),
-                    blobClient.getContainerName()
-                )
+        if (postLeaseCondition.isMet()) {
```

Comment:
Now that lease management is done at a higher level, `postLeaseCondition` can change name to `processingCondition` or sth. Up to you.